libs/go/zmssvctoken/signer.go

```diff
@@ -40,43 +40,52 @@ func hashString(input string) ([]byte, error) {
 	return h.Sum(nil), nil
 }
 
-// NewSigner creates an instance of Signer using the given private key (ECDSA or RSA).
-func NewSigner(privateKeyPEM []byte) (Signer, error) {
+// ExtractSignerInfo extract crypto.Signer and x509.SignatureAlgorithm from the given private key (ECDSA or RSA).
```

Comment:
is it possible to move this method to the athenzutils go library. it's somewhat confusing to use a function from zms named module in zts utils. that would provide better visibility and more common usage

Reply:
updated